embedded-graphics/src/lib.rs

```diff
@@ -100,15 +101,25 @@
 //! primitives and text. Converting the example above, we get this:
 //!
 //! ```rust
-//! use embedded_graphics::prelude::*;
-//! use embedded_graphics::pixelcolor::Rgb565;
 //! use embedded_graphics::fonts::Font6x8;
-//! use embedded_graphics::{egtext, egcircle};
+//! use embedded_graphics::pixelcolor::Rgb565;
+//! use embedded_graphics::prelude::*;
+//! use embedded_graphics::{egcircle, egtext};
 //! # use embedded_graphics::mock_display::MockDisplay;
 //! # let mut display = MockDisplay::default();
 //!
-//! let c = egcircle!((20, 20), 8, fill_color = Some(Rgb565::RED));
-//! let t = egtext!("Hello Rust!", Point::zero(), font = Font6x8, text_color = Some(Rgb565::GREEN)).translate(Point::new(20, 16));
+//! let c = egcircle!(
+//!     center = (20, 20),
+//!     radius = 8,
+//!     fill_color = Some(Rgb565::RED)
+//! );
+//! let t = egtext!(
+//!     text = "Hello Rust!",
+//!     top_left = Point::zero(),
+//!     font = Font6x8,
+//!     text_color = Some(Rgb565::GREEN)
+//! )
+//! .translate(Point::new(20, 16));
```

Comment:
If this isn't supposed to demonstrate `translate` the `top_left` argument shoud be set to `(20, 16)` instead.